media/js/firefox/os/firefox-os.js

```diff
@@ -59,7 +59,14 @@
             $provider.show();
 
             // setup GA event tracking on telecom provider exit links
-            $('#provider-links a').on('click', trackProviderExit);
+            $('#provider-links a').each(function() {
```

Comment:
Again, this file isn't even being used in production and is soon to be removed. I would suggest we don't bother changing it.